apps/mark-scan/frontend/src/pages/admin_screen.tsx

```diff
@@ -24,23 +24,22 @@ import {
   PollsState,
   PrecinctSelection,
 } from '@votingworks/types';
-import { makeAsync } from '@votingworks/utils';
 import { Logger } from '@votingworks/logging';
 import type { MachineConfig } from '@votingworks/mark-scan-backend';
 import type { UsbDriveStatus } from '@votingworks/usb-drive';
 import { ScreenReader } from '../config/types';
 import {
   ejectUsbDrive,
-  getPrecinctSelection,
   logOut,
   setPrecinctSelection,
+  setTestMode,
 } from '../api';
 
 export interface AdminScreenProps {
+  appPrecinct?: PrecinctSelection;
   ballotsPrintedCount: number;
   electionDefinition: ElectionDefinition;
-  isLiveMode: boolean;
-  toggleLiveMode: VoidFunction;
+  isTestMode: boolean;
```

Comment:
Do we have a preference for whether often-queried values are queried at each component or passed down from `app_root`? Seems like we are passing almost the entire `ElectionState` object down in props. When do we want to query from the subcomponent vs pass down as props?

Reply:
I would lean toward keeping the changes in this PR as-is because of consistency with `mark`, but I would also think that generally it would be better to move the query down a level rather than pass as props. 

The main disadvantage of this is that you have to insert loading behavior for subcomponents you wouldn't have to otherwise, when the needed data is coming through props. I avoided it for that reason! But you could also just make an assertion that the queries are not empty, making that disadvantage moot.